pkg/views/stat/list/view.go

```diff
@@ -0,0 +1,62 @@
+package list
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/charmbracelet/bubbles/table"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/goharbor/go-client/pkg/sdk/v2.0/models"
+	"github.com/goharbor/harbor-cli/pkg/views/base/tablelist"
+)
+
+var column = []table.Column{
+	{Title: "Public Project", Width: 14},
+	{Title: "Public Repo", Width: 12},
+	{Title: "Private Project", Width: 14},
+	{Title: "Private Repo", Width: 12},
+}
+
+var columnWide = []table.Column{
+	{Title: "Public Project", Width: 14},
+	{Title: "Public Repo", Width: 12},
+	{Title: "Private Project", Width: 14},
+	{Title: "Private Repo", Width: 12},
+	{Title: "Total Project Count", Width: 16},
+	{Title: "Total Repo Count", Width: 16},
+	{Title: "Total Storage Consumption", Width: 16},
+}
+
+func ListStatistics(stat *models.Statistic, wide bool) {
```

Comment:
Instead resprensenting it in table represent like this ,

You can play with theme also with charm.sh
![image](https://github.com/goharbor/harbor-cli/assets/136564604/0916ffa4-5416-4379-987b-2e5ac61eeb24)